README.rst

```diff
@@ -24,14 +24,30 @@ Moved to settings_.
 Basic Commands
 --------------
 
+Running locally with Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open a terminal at the project root and run the following for local development::
+
+    $ docker-compose -f local.yml up
```

Comment:
@Agrendalath, passing the `-f local.yml` argument can quickly become cumbersome. So it might be a good idea to mention here that the `COMPOSE_FILE` environment variable can be set to get rid of it in every command. This is mentioned in the documentation linked below, but imho this is handy enough to mention it here itself.

Also, it is the standard Django development server port, it is probably a good idea to mention that the web application is accessible at http://localhost:8000